tests/network-tests/src/fixtures/content/activeVideoCounters.ts

```diff
@@ -0,0 +1,383 @@
+    await this.assertCounterMatch('videoCategories', videoCategoryIds[1], oneMovedVideoCount)
+
+    /** Giza doesn't support changing channels - uncoment this on later releases where it's \\\
+
+    // move one video to another channel
+
+    this.debug('Move video to different channel')
+    await this.cli.updateVideo(videosData[0].videoId, {
+      channel: channelIds[1], // move from channel 1 to channel 2
+    })
+
+    // check counter of channel with newly moved video
+
+    this.debug('Checking channels active video counters (2)')
+    await this.assertCounterMatch('channels', channelIds[0], videoCount - oneMovedVideoCount)
+    await this.assertCounterMatch('channels', channelIds[1], oneMovedVideoCount)
+
+    // end
+    */
+
+    this.debug('Done')
+  }
+
+  /**
+    Prepares storage requisites.
+  */
+  private async prepareAssetStorage(storageLeader: Worker, storageGroupWorker: IMember) {
+    const noLimit = 10000000 // virtually no limit
+    const bucketSettings = {
+      sizeLimit: noLimit,
+      objectsLimit: noLimit,
+    }
+    const storageBucketsPerBag = 10 // something in boundaries of StorageBucketsPerBagValueConstraint (see runtime)
+
+    this.debug('Setting up storage buckets per bag limit')
+    await this.api.updateStorageBucketsPerBagLimit(storageLeader.role_account_id.toString(), storageBucketsPerBag)
+
+    this.debug('Setting up storage buckets voucher limits')
+    await this.api.updateStorageBucketsVoucherMaxLimits(
+      storageLeader.role_account_id.toString(),
+      bucketSettings.sizeLimit,
+      bucketSettings.objectsLimit,
+    )
+
+    this.debug('Adding worker to content directory group')
+    const workerId = await addWorkerToGroup(
+      this.api,
+      this.env,
+      WorkingGroups.Storage,
+      storageGroupWorker.keyringPair.address
+    )
+
+    this.debug('Creating storage bucket')
+    const createBucketResult = await this.api.createStorageBucket(
+      storageLeader.role_account_id.toString(),
+      bucketSettings.sizeLimit,
+      bucketSettings.objectsLimit,
+      workerId,
+    )
+
+    const storageBucketId = this.api.findStorageBucketCreated(createBucketResult.events) as DataObjectId
+
+    this.debug('Accepting storage bucket invitation')
+    await this.api.acceptStorageBucketInvitation(
+      storageGroupWorker.keyringPair.address,
+      workerId,
+      storageBucketId,
+    )
+
+    return {workerId, storageBucketId}
+  }
+
+  /**
+    Asserts a channel, or a video/channel categories have their active videos counter set properly
+    in Query node.
+  */
+  private async assertCounterMatch(
+    entityName: 'channels' | 'channelCategories' | 'videoCategories',
+    entityId: number,
+    expectedCount: number
+  ) {
+    const qnConnectionNumberOfRepeats = 10
+
+    const getterName = `get${entityName[0].toUpperCase()}${entityName.slice(1)}`
+    await this.query.tryQueryWithTimeout(
+      () => (this.query as any)[getterName](),
```

Comment:
Merging `giza_staging` will allow you to use typesafe queries (https://github.com/Joystream/joystream/blob/giza_staging/tests/network-tests/src/QueryNodeApi.ts)

Reply:
Fixed. Cast to `any` no longer needed there thx to typecasting `getterName`
````
const getterName = `get${entityName[0].toUpperCase()}${entityName.slice(1)}` as 'getChannels' | 'getChannelCategories' | 'getVideoCategories'
````